src/latex/devcontainer-feature.json

```
@@ -24,11 +24,11 @@
         },
         "mirror": {
             "type": "string",
-            "default": "https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz",
+            "default": "https://mirror.ctan.org/systems/texlive/tlnet/",
             "description": "Provide a custom mirror to use for the installation of Tex Live. Omit option to automatically select the mirror closest to you.",
             "proposals": [
-                "https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz",
-                "https://mirrors.mit.edu/CTAN/systems/texlive/tlnet/install-tl-unx.tar.gz"
+                "https://mirror.ctan.org/systems/texlive/tlnet/",
+                "https://mirrors.mit.edu/CTAN/systems/texlive/"

```

the trailing directory is needed to properly download install-tl-unx.tar.gz from mirror

```suggestion
                "https://mirrors.mit.edu/CTAN/systems/texlive/tlnet"
```